tests/aio/test_types.py

```diff
@@ -29,7 +29,7 @@
         ('{"foo":"bar"}', "JsonDocument"),
         (uuid4(), "Uuid"),
         ([1, 2, 3], "List<Int8>"),
-        # ({1, 2, 3}, "Set<Int8>"),  # FIXME: AttributeError: 'set' object has no attribute 'items'
+        ({1: None, 2: None, 3: None}, "Set<Int8>"),
```

Comment:
Is it difficult to fix the scenario for set too?

Reply:
It's not difficult to transfer a set, but in this case on the receive stage we won't know for sure whether it was a set or a dictionary with empty values (Dict<Utf8, Void> for example). And if we pass a set and get a dictionary, then we get inconsistent behavior.